src/adapters/kafka/index.ts

```diff
@@ -44,17 +43,22 @@ class KafkaAdapter extends Adapter {
         this.emit('message', msg, consumer)
       },
     })
-    
+  }
+
+  async send(message: GleeMessage) {
     const producer = kafka.producer()
```

Comment:
`kafka` is not defined here. It's in the `connect` function. My suggestion is that, in the `connect` function, you assign it to `this.kafka` instead of doing `const kafka`. This way, the variable will be available on the whole class using `this.kafka`.